pushover.sh

```diff
@@ -93,11 +96,28 @@ message="$*"
 if [ ! -x "${CURL}" ]; then
     echo "CURL is unset, empty, or does not point to curl executable. This script requires curl!" >&2
     exit 1
+else
```

Comment:
This doesn't need to be in an else block: The program terminates if curl is not found